io-backend/src/main/java/agh/io/iobackend/model/GameRoom.java

```diff
@@ -20,35 +23,70 @@ public class GameRoom {
     )
     private Long gameRoomID;
 
-    @Column(
-            name = "gameMasterID",
-            nullable = false
-    )
+    @Column(name = "gameMasterID",
+            nullable = false)
     private Long gameMasterID;
 
-    @Column(
-            name = "mapID"
-    )
+    @Column(name = "mapID")
     // TODO mapID will be the foreign key for maps database, where details about the map are stored
     private Long mapID;
 
-    @Column(
-            name = "roomCode",
-            unique = true,
-            nullable = false
-    )
-    // Game code exchanged by players in order to join the play
-    private String roomCode;
```

Comment:
Dlaczego rezygnujemy z roomCode?

Reply:
Znaczy ja nie wiem, czy rezygnujemy, ja go na razie po prostu nie będę chyba implementować, jak coś to można później myślę.